lib/player.rb

```diff
@@ -0,0 +1,9 @@
+class Player
```

Comment:
Lets talk about this. Im not sure we want a Player class that has its own Board. I think we can do this all from the Game class (since they share the same Board)